### PR TITLE
The console is now toggleable

### PR DESCRIPTION
**Pull request by YDutchy**

### CODE REVIEW

File: src/main/resources/Basic_Gui.fxml [hunk 1]
```diff
@@ -38,14 +39,14 @@
                    <TextField fx:id="txtCenterNode" layoutX="20.0" layoutY="251.0" minWidth="100.0" promptText="Origin node" AnchorPane.leftAnchor="20.0" AnchorPane.rightAnchor="20.0" />
                    <Button fx:id="btnDrawRandom" layoutX="20.0" layoutY="350.0" minWidth="100.0" mnemonicParsing="false" text="Surprise me!" />
                </children></AnchorPane>
-            <SplitPane dividerPositions="0.6951" orientation="VERTICAL" prefHeight="200.0" prefWidth="160.0">
+            <SplitPane dividerPositions="1.0" orientation="VERTICAL" prefHeight="200.0" prefWidth="160.0">
```
Comment (toinehartman) [inline src/main/resources/Basic_Gui.fxml]: You should remove the SplitPane if we do not split the view anymore.

File: src/main/resources/Basic_Gui.fxml [hunk 2]
```diff
@@ -38,14 +39,14 @@
...
               <items>
                 <AnchorPane fx:id="anchorGraphPanel" minHeight="200" minWidth="600" prefHeight="400" prefWidth="600">
                      <children>
                          <Group fx:id="grpDrawArea" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0" />
                      </children>
                   </AnchorPane>
-                <AnchorPane fx:id="anchorConsolePanel" minHeight="100" minWidth="300" prefHeight="100.0" prefWidth="600" />
+                <AnchorPane fx:id="anchorConsolePanel" minHeight="0.0" minWidth="0.0" prefHeight="0.0" prefWidth="600.0" />
```
Comment (toinehartman) [inline src/main/resources/Basic_Gui.fxml]: Please remove this as well.

File: src/main/java/programminglife/gui/controller/GuiController.java [hunk 3]
```diff
@@ -341,24 +344,42 @@ private void initMouse() {
...
-        AnchorPane.setBottomAnchor(console, 0.d);
-        AnchorPane.setTopAnchor(console, 0.d);
-        AnchorPane.setRightAnchor(console, 0.d);
-        AnchorPane.setLeftAnchor(console, 0.d);
+        AnchorPane root = new AnchorPane();
+        btnToggle.setSelected(false);
+        console.setVisible(false);
+        root.setVisible(false);
+        Stage st = new Stage();
+        st.setScene(new Scene(root, 500, 500, Color.GRAY));
```
Comment (toinehartman) [inline src/main/java/programminglife/gui/controller/GuiController.java]: Please set a minimum width for the console window (for example 200x100 at least).

Comment (YDutchy) [inline src/main/java/programminglife/gui/controller/GuiController.java]: I have now set a minimum for the console size and changed the FXML accordingly

**Pull request merged**